cpp/src/io/parquet/stats_filter_helpers.cpp

```diff
@@ -25,20 +25,21 @@
 
 namespace cudf::io::parquet::detail {
 
-stats_expression_converter::stats_expression_converter(ast::expression const& expr,
-                                                       size_type num_columns)
-  : _num_columns{num_columns}
+stats_columns_collector::stats_columns_collector(ast::expression const& expr,
```

Comment:
Moved as is from `stats_expression_converter` to the new base class `stats_columns_collector`